src/docfx/publish/PublishItem.cs

```diff
@@ -30,6 +30,8 @@ internal class PublishItem
     /// </summary>
     public string? SourcePath { get; init; }
 
+    public string? ContentType { get; init; }
```

Comment:
Should it nullable?

Reply:
I notice all the property under this class are nullable, @AngryBerryMS do you know the background?